source/extract_combine_patches.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>is this crash expected ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
The errors are intentional. They are meant to show when the process fails and the reason for doing so.

Reply:
<p>This particular one is to show that the original image size is expected to be divisible by 2. The next cells show how to add padding to remedy this issue.</p>

Reply:
Removed errors since the might be difficult to test.